Content.Server/Atmos/Piping/EntitySystems/AtmosUnsafeUnanchorSystem.cs

```diff
@@ -30,15 +31,12 @@ private void OnUnanchorAttempt(EntityUid uid, AtmosUnsafeUnanchorComponent compo
             if (!component.Enabled || !EntityManager.TryGetComponent(uid, out NodeContainerComponent? nodes))
                 return;
 
-            if (_atmosphere.GetContainingMixture(uid, true) is not {} environment)
-                return;
-
             foreach (var node in nodes.Nodes.Values)
             {
                 if (node is not PipeNode pipe)
                     continue;
 
-                if (pipe.Air.Pressure - environment.Pressure > 2 * Atmospherics.OneAtmosphere)
+                if (pipe.Air.Pressure > Atmospherics.OneAtmosphere)
```

Comment:
Perhaps this check should stay the same -- because it's the pressure difference between pipe and environment that actually matters, and not the absolute pressures. The proposed change would sometimes warn when there is no pressure difference risk.

Also perhaps good to keep 2 ATM instead of 1.

Reply:
It is the absolute pressure that matters now, since that decides how many moles of gas are inside the pipe and will leak out when unanchored. This amount will be the same, no matter the outside pressure.

Changing it to 2 ATM would be reasonable.

Reply:
I see what you're saying. But I think the old intent was something akin to SS13's warning when unwrenching pipes because the space wind would fling you out if you didn't have magboots on. That is why unwrenching a pipe is "unsafe", not because it would leak gases.

Now, we don't have space wind or the flinging mechanic so that's perhaps that's a case to change the behavior of this warning. But I kind of do like the warning being based on pressure differences -- this is something you could actually theoretically feel when starting to unwrench such a pipe. Otherwise, you're just kind of magically sensing what's inside the pipe.

Up to you what you want to do about this.

Reply:
Oh, I think I get your point. The warning is not about the amount of gas removed but about its impact on the environment it is realeased into. In vacuum 16 moles is a lot, wheras in a 4500kPA it barely changes the relative gas mix.

I will change it back later today.